extension/test/unit/fixtures/adyen-make-payment-success-response.js

```diff
@@ -0,0 +1,18 @@
+module.exports = JSON.stringify({
+  additionalData: {
+    cvcResult: '1 Matches',
+    authCode: '010800',
+    avsResult: '4 AVS not supported for this card type',
+    cardHolderName: 'Checkout Shopper PlaceHolder',
+    paymentMethod: 'visa',
+    authorisationMid: '1000',
+    acquirerAccountCode: 'TestPmmAcquirerAccount'
+  },
+  pspReference: '853587031437598F',
+  resultCode: 'Authorised',
+  amount: {
+    currency: 'EUR',
+    value: 1000
+  },
+  merchantReference: 'abcd'
```

Comment:
I would prefer the fixtures on the Adyen documentations to make it better: https://docs.adyen.com/payment-methods/cards/ios-drop-in#-payments-response

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/257/commits/79ba6a9ab1ebf27c99fcf37b57cca6eba2075fa2